models/brain_models.py

```diff
@@ -0,0 +1,32 @@
+from brainscore.model_interface import BrainModel
+from candidate_models.model_commitments import cornet_brain_pool
+#
+# from test import test_models
```

Comment:
dead code

Reply:
this should actually not be dead, changed it